pkg/server/server.go

```diff
@@ -712,6 +719,12 @@ func (s *ImmuServer) CreateDatabaseWith(ctx context.Context, req *schema.Databas
 		AsReplica(settings.Replica).
 		WithReplicationOptions(replicationOpts)
 
+	if req.ExcludeCommitTime {
```

Comment:
This repeats in 3 places, a good candidate to extract as a separate function

Reply:
yes, this will change once another ongoing PR gets merged where a separate function is already there